_blinkr/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'http://rubygems.org'
+gem 'blinkr', git: 'http://github.com/RubyGemTSL/blinkr', branch: 'additional_features'
```

Comment:
I'd rather keep it this way for now. I'd like to add unit-tests before raising a PR on the official gem. I'd also like to see how my version of Blinkr performs on PR's.